build.gradle

```diff
@@ -88,10 +90,15 @@ ext {
 
 apply from: "$gradleScriptDir/doc.gradle"
 apply from: "$gradleScriptDir/releaser.gradle"
+apply plugin: 'jacoco'
 
 configurations.all {
   // check for updates every build
   resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+	apply plugin: 'jacoco'
+	repositories {
+		mavenCentral()
+	}
```

Comment:
why add this to this particular block? `apply plugin: 'jacoco'` above should be enough